src/main/scala/com.socrata.snapshotter/BlobStoreManager.scala

```diff
@@ -76,10 +128,10 @@ object BlobStoreManager {
   class LoggingListener(private val logger: org.slf4j.Logger) extends ProgressListener {
     def progressChanged(event: ProgressEvent): Unit = {
       if (event.getEventType == com.amazonaws.event.ProgressEventType.TRANSFER_COMPLETED_EVENT) {
-        logger.info("Upload complete.")
+        logger.debug("Upload complete.")
```

Comment:
Unless we're running at debug log level everywhere I think this and the start log should be at least the level we run at in production.